charger/easee.go

```diff
@@ -424,12 +426,24 @@ func (c *Easee) Enable(enable bool) error {
 	if err != nil {
 		return err
 	}
-	if noop {
-		return c.confirmChargerCurrent(targetCurrent)
-	}
 
-	if err := c.waitForDynamicChargerCurrent(targetCurrent); err != nil {
-		return err
+	if noop {
+		c.mux.Lock()
+		enabled := c.inEnabledOpMode()
```

Comment:
Hier hast Du einen Deadlock mit 2x Mutex lock. Den Case kannst Du nicht getestet haben ;) Wäre gut das nochmal sorgfältig in allen Fällen durchzuspielen.